protos/flyteidl/core/tasks.proto

```diff
@@ -121,6 +121,7 @@ message TaskTemplate {
     oneof target {
         Container container = 6;
         K8sPod k8s_pod = 17;
+        Sql sql = 18;
```

Comment:
is `Query` better sounding?

Reply:
SQL -> Structured query language. So SQL is fine IMO